cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -1294,11 +1294,11 @@ private String getHiveDatabase(@Nullable String namespace) {
    * @throws IOException
    * @throws ExploreException
    */
-  protected Map<String, String> startSession() throws IOException, ExploreException {
+  protected Map<String, String> startSession() throws Exception {
```

Comment:
schedulerQueueResolver.getQueue shouldn't throw Exception. It should only need to throw IOException.